inc/msp/FlightController.hpp

```diff
@@ -3,46 +3,92 @@
 
 #include "Client.hpp"
 #include "msp_msg.hpp"
+#include "flightmode.hpp"
+#include "periodic_timer.hpp"
 
 namespace fcu {
 
-enum class FirmwareType {
-    MULTIWII,
-    CLEANFLIGHT
+enum class ControlSource : uint8_t
```

Comment:
Does this enum need to be 8bit? I used a class enum with 8bit for the MSP IDs because they needed to be serialised for the protocol. If you do not require a certain type of enum, just use the regular `enum`.